pages/pipelines/trigger_step.md.erb

```diff
@@ -17,78 +17,96 @@ _Required attributes:_
   <tr>
     <td><code>trigger</code></td>
     <td>
-      The slug of the triggered pipeline. The pipeline slug must be lowercase.<br>
+      The slug of the pipeline to create a build. The pipeline slug must be lowercase.<br>
       <em>Example:</em> <code>"another-pipeline"</code>
     </td>
   </tr>
 </table>
 
-Optional attributes:
+_Optional attributes:_
 
 <table>
   <tr>
     <td><code>build</code></td>
     <td>
-      TODO
+      A map of attributes for the triggered build. See below for the list of attributes.
     </td>
   </tr>
   <tr>
     <td><code>label</code></td>
     <td>
-      TODO
+      The label that will be displayed in the pipeline visualisation in Buildkite. (supports emoji)<br>
+      <em>Example:</em> <code>"\:rocket\: Deploy"</code><br>
     </td>
   </tr>
   <tr>
     <td><code>async</code></td>
     <td>
-      TODO
+      A boolean value for whether the step should continue the pipeline without waiting for the triggered build to complete. If set to <code>true</code> the step will immediately continue, regardless of the success of the triggered build. If set to <code>false</code> it will wait for the triggered build to complete and continue only if the triggered build passed.
```

Comment:
Maybe doesn't need the first sentence. It's a bit run on and confusing as is, but the examples make it clear what you're trying to describe.